src/corona_hakab_model/analyzers/population_analysis.py

```diff
@@ -0,0 +1,85 @@
+from os import path
+from typing import Tuple
+import pickle
+import pandas as pd
+from generation.connection_types import ConnectionTypes
+import matplotlib.pyplot as plt
+from analyzers.config import POPULATION_OUTPUT_PATH
+
+
+def load_population_data_to_dfs() -> Tuple[pd.DataFrame, pd.DataFrame, pd.DataFrame]:
+    output_file_path = POPULATION_OUTPUT_PATH
```

Comment:
Please give the output_file_path
You can give the parameter POPULATION_OUTPUT_PATH as default value